packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -434,6 +434,40 @@ export class CdkToolkit {
     }
   }
 
+  /**
+   * Roll back the given stack or stacks.
+   */
+  public async rollback(options: RollbackOptions) {
+    const startSynthTime = new Date().getTime();
+    const stackCollection = await this.selectStacksForDeploy(options.selector, true);
+    const elapsedSynthTime = new Date().getTime() - startSynthTime;
+    print('\n✨  Synthesis time: %ss\n', formatTime(elapsedSynthTime));
+
+    if (stackCollection.stackCount === 0) {
+      // eslint-disable-next-line no-console
+      console.error('No stacks selected');
+      return;
+    }
+
+    for (const stack of stackCollection.stackArtifacts) {
+      print('Rolling back %s', chalk.bold(stack.displayName));
+      const startRollbackTime = new Date().getTime();
+      try {
+        await this.props.deployments.rollbackStack({
+          stack,
+          roleArn: options.roleArn,
+          toolkitStackName: options.toolkitStackName,
+          force: options.force,
+        });
+        const elapsedRollbackTime = new Date().getTime() - startRollbackTime;
+        print('\n✨  Rollback time: %ss\n', formatTime(elapsedRollbackTime));
+      } catch (e: any) {
+        error('\n ❌  %s failed: %s', chalk.bold(stack.displayName), e.message);
+        throw new Error('Rollback failed (use --force to orphan failing resources)');
```

Comment:
What do you mean?

Reply:
Oh so I think I was commenting from the position that if a stack cannot be rolled back, we should throw an error - in this case a single faulty stack can prevent rolling back others, so we need to accumulate errors. 

If you insist on no-oping for a unrollable stack, this is fine, but I still think we should error out.

Reply:
So I see that right now you do throw an error on `ROLLBACK_FAILED` state - doesn't this mean you need to swallow the error here and proceed? to avoid the poison pill?